packages/design-system/components/inputs/SocialTextField/types.ts

```
@@ -1,13 +1,16 @@
 
 import { TextFieldProps } from '../TextField/types'
+import { TextareaFieldProps } from '../TextareaField/types'
 
 export type SocialTextFieldProps = FormControl &
   TextFieldProps &
   PropsWithChildren & {
     isReply?: boolean
     onCancelReply?: () => void
     replyTargetName?: string | null
+    key?: string
+    TextField?: FC<TextareaFieldProps> | React.ComponentType<{}>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)